apps/fluent-tester/src/FluentTester/TestComponents/Shimmer/ShimmerTest.win32.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react';
+import { View, Text } from 'react-native';
+
+export const ShimmerTest: React.FunctionComponent = () => {
+  return (
+    <View testID="ScrollViewAreaForComponents">
+      <Text>{"Shimmer isn't supported on Win32."}</Text>
```

Comment:
This is not true, Shimmer is explicitly supported on win32. Will need to figure out why the type check didn't work...